systemd/user/adsys-user-mounts.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=ADSys user mount handler
+After=network-online.service
+ConditionPathExists=/run/adsys/users/%U/mounts
+ConditionPathExists=/usr/libexec/adsys_mount
```

Comment:
I don’t think the second ConditionPathExists really makes sense. We rarely have a service checking for the binaries it’s running if they exists or not.

If the service is installed, it’s expected for the binary to be present on disk, or it’s a packaging or system consistency issue.

Similarly, if a binary is using an asset (like an image), the fact that the image is present is a delivery and consistency issue, not a code one.

Making sense?

Reply:
It makes sense, indeed. But, even though the binary is supposed to be there, I felt like it would be good to have this condition explicitly defined in the unit for clarity. Wdyt?

Reply:
Actually, I would prefer not:
- as said, this is a system issue to not have the binary when you the service
- if we keep the ConditionPathExists, it means that the unit is skipped, not in failure, when it should rather fail :)

Reply:
Ack, I will fix it in #544.